cylc/uiserver/schema.py

```diff
@@ -98,6 +98,8 @@ class Play(graphene.Mutation):
     class Meta:
         description = sstrip('''
             Start, resume or restart a workflow run.
+
+            Valid for: stopped, paused workflows.
```

Comment:
I think it might be better to force the "resume" use cases to go through the `resume` interface as this avoids a subprocess call and a bunch of config which doesn't apply to resume cases:

```suggestion
            Valid for: stopped workflows.
```